soc/xlnx/versalnet/Kconfig.soc

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_AMD_VERSALNET
+	bool
+
+config SOC_AMD_VERSALNET_RPU
+	bool
+	select SOC_AMD_VERSALNET
+	help
+	  AMD Versal NET SoC
+
+config SOC_FAMILY
+	default "amd_versalnet" if SOC_AMD_VERSALNET
+
+config SOC
+	default "versalnet_rpu" if SOC_AMD_VERSALNET_RPU
```

Comment:
name mismatch either needs to be `default "amd_versalnet_rpu" if SOC_AMD_VERSALNET_RPU` or `default "versalnet_rpu" if SOC_VERSALNET_RPU`, and soc.yml needs to be the same

Reply:
sure will fix